StepDefinitions/NewUser.js

```diff
@@ -0,0 +1,49 @@
+const {
+    Given,
+    When,
+    Then,} = require('cucumber');
+
+const Functions = require('../Functions.js');
+
+let credentials
+
+Given('I want to create new user',async(dataTable) => {
+    credentials = dataTable.hashes();
+    await Functions.createNewUser(credentials);
+});
+
+
+Then('A new user {string} should be created',async(username) => {
+    await Functions.getUser(username,credentials);
+
+});
+
+
+When('A user {string} updates email to {string}',async(username,email) => {
+    await Functions.emailUpdate(username,email,credentials);
+});
+
+
+Then('His email should be updated to {string}',async(email) => {
+    await Functions.checkEmail(email,credentials);
+});
+
+
+Then('A user should be able to login successsfully',async() => {
+   await Functions.loginUser(credentials[0].username, credentials[0].password);
```

Comment:
you are sending username and password but your loginUser() function is receiving just credentials - you should fix that